tic-tac-toe.js

```diff
@@ -1,7 +1,221 @@
 function TicTacToe() {
+  this.board = [["","",""], ["","",""], ["","",""]];
+  this.count = 0;
+  }
 
-}
+  var checkSpot = function(col, row, sign , board) {
+    if (board[col][row] === sign) {
+      return true;
+    } else {return false;}
+  }
 
-TicTacToe.prototype = {
-  
-}
+TicTacToe.prototype.CheckRow = function(col, row, sign) {
+    if (row === 0) {
+      if (checkSpot(col, 1, sign, this.board) && (checkSpot(col,2, sign, this.board))) {
+        return true;
+    } else if ( row === 1) {
+      if (checkSpot(col, 0, sign, this.board) && (checkSpot(col,2, sign, this.board)))
+        return true;
+    } else {
+      if (checkSpot(col, 0, sign, this.board) && (checkSpot(col,1, sign, this.board)))
+        return true;
+    }
+  }
+};
+
+TicTacToe.prototype.CheckCol = function(col, row, sign) {
+  if (col === 0) {
+    if (checkSpot(1,row, sign, this.board) && (checkSpot(2,row, sign, this.board)))
+      return true;
+  } else if ( col === 1) {
+    if (checkSpot(0, row, sign, this.board) && (checkSpot(2,row, sign, this.board)))
+      return true;
+  } else {
+    if (checkSpot(0, row, sign, this.board) && (checkSpot(1,row, sign, this.board)))
+      return true;
+  }
+};
+
+TicTacToe.prototype.checkOblique = function(col, row, sign) {
+  if ((col === 0) && (row == 1) || (col == 1 && row === 0) || (col == 1 && row == 2) || (col == 2 && row == 1) ) {
+    return false;
+  } else if ((checkSpot(0,0,sign, this.board) && checkSpot(1,1,sign, this.board) && checkSpot(2,2, sign, this.board)) || (checkSpot(0,2,sign, this.board) && checkSpot(1,1,sign, this.board) && checkSpot(2,0,sign, this.board))){
+    return true;
+  }
+};
+
+TicTacToe.prototype.checkAvai = function(col, row) {
+  if (this.board[col][row] === "") {
+    return true;
+  } else { return false;}
+};
+
+  $(document).ready(function() {
+    var game = new TicTacToe();
+
+    $("#zz").click(function(){
```

Comment:
I think that this duplicate logic only works because there are 9 spots, but if you added even one additional row/col it would be even more difficult. I would really like to see you try to refactor this to use a single click event handler rather that an individual one for each space.